src/applications/vaos/appointment-list/components/cards/confirmed/VideoLink.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import classNames from 'classnames';
+import moment from 'moment';
+import {
+  getVideoKind,
+  isVideoGFE,
+  isAtlasLocation,
+} from '../../../../services/appointment';
+import { VIDEO_TYPES } from '../../../../utils/constants';
+
+export default function VideoLink({ appointment }) {
```

Comment:
Extracted from logic that created the `linkContent` variable in VideoVisitSection